package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "packageManager": "pnpm@9.1.3",
   "scripts": {
-    "typecheck": "tsc --noEmit",
+    "typecheck": "tsc",
```

Comment:
Putting noEmit inside the `tsconfig.json` is a best practice, since users of this repo won't always know to run `pnpm typecheck` and not just `pnpm tsc`.